package.json

```diff
@@ -89,6 +89,7 @@
     "glob": "5.0.15",
     "google-libphonenumber": "3.2.3",
     "html-webpack-plugin": "2.22.0",
+    "isemail": "2.2.1",
```

Comment:
Curious why you chose this version and not the most recent. Though maybe you'll have to change this because travis is unhappy?

Reply:
Yeah, i was trying to get Travis to be happy, but it turns out it's not Travis's fault! I'm going to try a different package instead.

Reply:
Used another package, working better!